lib/cisco_node_utils/cmd_ref/feature.yaml

```diff
@@ -18,6 +18,11 @@ fabric_forwarding:
   config_get_token: '/^feature fabric forwarding$/'
   config_set: "feature fabric forwarding"
 
+fabric_frwd_anycast_mac:
```

Comment:
Is this supported on all platforms? It doesn't seem to be on my 9k or 5k. Please add an `_exclude:` for the applicable platforms.